package.json

```diff
@@ -132,7 +132,11 @@
         "webpack": "^5.82.1",
         "webpack-bundle-analyzer": "^4.8.0",
         "webpack-cli": "^5.1.1",
-        "webpack-dev-server": "^4.15.0"
+        "webpack-dev-server": "^4.15.0",
+        "@semantic-release/changelog": "^6.0.3",
+        "@semantic-release/github": "^9.2.6",
+        "@semantic-release/npm": "^11.0.3",
+        "@semantic-release/release-notes-generator": "^12.1.0"
```

Comment:
since we have semantic release can we set the package version to 0.0.0-semantic just to be clear?